js/google_ima.js

```diff
@@ -1385,6 +1385,17 @@ require("../html5-common/js/utils/utils.js");
        */
       var _onImaAdError = privateMember(function(adError)
       {
+        if (adError &&
+          adError.type === 'adError' &&
+          adError.h &&
```

Comment:
Is this safe? It looks like you are accessing a the variable from the minified code which could change with their next release. There should be a function you can call to extract the ad error code.

Reply:
https://developers.google.com/interactive-media-ads/docs/sdks/html5/v3/apis#ima.AdError

just do aderror.getErrorCode()

Reply:
Thank you, Peter. Fixed this.